common/src/main/java/com/wynntils/utils/mc/LoreUtils.java

```diff
@@ -1,9 +1,11 @@
 /*
- * Copyright © Wynntils 2021.
+ * Copyright © Wynntils 2021-2023.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.utils.mc;
 
+import static com.wynntils.models.wynnitem.parsing.WynnItemParser.SHINY_STAT_NO_CAPTURE_PATTERN;
```

Comment:
We disallow static imports, please remove them from the code.

Reply:
okay, got it